joml-geometry/src/main/java/org/terasology/joml/geom/Rectanglef.java

```diff
@@ -380,11 +407,9 @@ public boolean isValid() {
     /**
      * Compute the rectangle of intersection between <code>this</code> and the given rectangle.
      * <p>
-     * If the two rectangles do not intersect, then {@link Float#NaN} is stored in each component
-     * of <code>dest</code>.
+     * If the two rectangles do not intersect, then {@link Float#NaN} is stored in each component of <code>dest</code>.
```

Comment:
There is no `dest` is there?

Reply:
auto-formatting :roll_eyes: I will not go over the unrelated docstrings in this PR.